app/config/collections/common.php

```diff
@@ -867,6 +867,138 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('continentCode'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 2,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('latitude'),
+                'type' => Database::VAR_FLOAT,
+                'format' => '',
+                'size' => 2,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('longitude'),
+                'type' => Database::VAR_FLOAT,
+                'format' => '',
+                'size' => 2,
```

Comment:
Size 2 sounds scarry. Lets check with Smuel, maybe few extra bites wont hurt too much, and prevents possible problems in future